scripts/deltapreview/deltapreview.js

```diff
@@ -0,0 +1,113 @@
+/*
+Delta Previewer by Corviknight [https://pfq.link/@Kp]
+Adds an option to the Pokemon menu to preview a different Delta type or remove Delta type.
+Due to an update to Delta image URLs/handling on 6/21/25, not all possible combinations are populated on the server yet, and it may return a 404 until one is hatched or is viewed naturally on the site. 
+This is a known limitation, but the issue will lessen as more Delta combinations are viewed or discovered.
+*/
+
+(function() {
+    'use strict';
```

Comment:
```suggestion
    'use strict';
    const allowedPaths = [
        /^\/summary\/[^/]+$/,
        /^\/party$/
    ];

    if (!allowedPaths.some(pattern => pattern.test(location.pathname))) return;
```

I know I just approved this but after discussion, we've added it as a requirement to only run where the script is needed. More details [here](https://github.com/pokefarm-com/userscripts/wiki).